Sources/xcodeproj/Objects/Files/PBXContainerItemProxy.swift

```diff
@@ -1,5 +1,11 @@
 import Foundation
 
+public enum ContainerPortal {
```

Comment:
I'd move this one into `PBXContainerItemProxy`

Reply:
@pepibumur this is done